canopen_402/src/motor.cpp

```diff
@@ -504,6 +508,8 @@ void Motor402::handleInit(LayerStatus &status){
         return;
     }
 
+    homing->setHomingTimeout(homing_timeout_);
```

Comment:
Why did you create a new function and did not  pass it as a parameter to `executeHoming`

Reply:
@ipa-mdl tbh, I didn't know which option was better, I'll refactor to have it as:
`executeHoming(canopen::LayerStatus &status, const boost::chrono::seconds &homing_timeout)`
and pass it with the call to `executeHoming`

Reply:
Me neither ;)

That's why I wanted to know why you chose that option.

> I'll refactor to have it as:
executeHoming(canopen::LayerStatus &status, const boost::chrono::seconds &homing_timeout)
and pass it with the call to executeHoming

:+1:

Please keep the function without the argument as well (stable API) and call the new one with a timeout of 10s (as before).
I will try to figure out, if we can avoid the ABI break in `Motor402`.

Reply:
I've added the new function and replace the call to `executeHoming` with a default timeout of 10s